src/components/projects/ProjectDetails/ProjectDetails.scss

```diff
@@ -18,6 +18,87 @@
   position: relative;
   font-size: 0.7rem;
 
+  .fa-arrow-circle-right {
+    position: absolute;
+    top: -16px;
+    left: 6%;
+    cursor: pointer;
+  }
+
+  .parent-name-span {
+    position: absolute;
+    top: -10px;
+    left: 90px;
+    color: #000;
+    font-weight: 700;
+    word-break: break-all
+  }
+
+  .empty-project-phases-container {
+    width: 100%;
+    min-height: 270px;
+    margin: 25px auto;
+    background-color: rgba(0, 0, 0, 0.2);
+    border-radius: 20px;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    @media(min-width: 764px){
```

Comment:
That is good to use 'screen' atribute. coz IE do not recognize only media query (without screen atribute)
https://www.w3schools.com/cssref/css3_pr_mediaquery.asp